routes/index.js

```diff
@@ -215,6 +216,27 @@ router.get('/admin', isLoggedIn, function (req, res, next) {
   res.render('pages/admin-console.ejs')
 })
 
+router.get('/admin-all-registrants', isLoggedIn, function(req, res, next) {
+  if(req.user.local.email !== process.env.ADMIN_EMAIL) {
+    res.redirect('/')
+    return
```

Comment:
do we need a return statement here?

Reply:
I think the return escapes the block, otherwise the `.getAllUsers` function would be called and an error thrown for `res.json` after response sent. 

But I could be wrong...

Reply:
@KenG98 can we make it `return res.redirect('/')`? @lkwatson is right

Reply:
same goes for the end of all routes - make sure to `return res....` unless it's the only thing in the route, otherwise express will throw an error saying you sent two responses for the same thing